src/main.py

```diff
@@ -1,9 +1,20 @@
-# Resolve the problem!!
+import re
+import string
 
 
-def run():
-    # Start coding here
 
+def run():
+    
+    with open('encoded.txt', 'r', encoding='utf-8') as f:
+        message= f.readline()
+    
+    
+    hidden=""   
+    for f in re.findall("([a-z]+)", message):
```

Comment:
Hola David, 

Una solucion para evitar el ciclo for es usar el metodo join para unir la lista

Reply:
Claro Andrés. Gracias!!!